libcamera/build.rs

```diff
@@ -6,11 +6,7 @@ use std::{
 use semver::{Comparator, Op, Version};
 
 fn main() {
-    let libcamera_version = Version::new(
-        libcamera_sys::LIBCAMERA_VERSION_MAJOR as _,
-        libcamera_sys::LIBCAMERA_VERSION_MINOR as _,
-        libcamera_sys::LIBCAMERA_VERSION_PATCH as _,
-    );
+    let libcamera_version = Version::new(0, 3, 2);
```

Comment:
hi if your doing this so you can cross compile i hope https://github.com/lit-robotics/libcamera-rs/pull/48 should help, could you try it yourself to confirm?